app/src/main/java/io/gnosis/safe/ui/base/BaseFragment.kt

```diff
@@ -17,9 +18,9 @@ abstract class BaseFragment<T> : Fragment()
     private var _binding: T? = null
     protected val binding get() = _binding!!
 
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        inject(buildViewComponent())
+    override fun onAttach(context: Context) {
```

Comment:
This can be potentially called multiple times, is this an assue for dagger?

Reply:
Also recommended by the documentation: 

> When using fragments, inject Dagger in the fragment's onAttach() method. In this case, it can be done before or after calling super.onAttach().

Same link as in the previous comment.